specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -1203,7 +1203,7 @@
         },
         "enableSoftDelete": {
           "type": "boolean",
-          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If omitted, assume true as default value. Once set to true, cannot be reverted to false."
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. Once it's been set to true value, it can NOT be reverted to false."
```

Comment:
"Once set to true, it cannot be reverted to false."

Reply:
I copied the text from `VaultPatchProperties.enableSoftDelete`

https://github.com/Azure/azure-rest-api-specs/blob/4e68c00b1195e3eb1f1acf62ae2b0c267883e222/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json#L1123-L1127

Not sure why it was changed there. So just want keep both in sync, but I actually prefer the old one. Do you want me to change `VaultPatchProperties.enableSoftDelete` too?

Reply:
I would change the other one as well. "cannot" is one word, and "set to true value" is not grammatically correct. Since this generates public documentation, we should clean it up a bit.

Reply:
Thanks @heaths. Changed as requested.